.pre-commit-config.yaml

```diff
@@ -40,3 +40,11 @@ repos:
         args:
           - --exclude-charts=alfresco-common,alfresco-sync-service
           - --values-dir=ci
+  - repo: local
+    hooks:
+    -   id: update-charts-yml
+        name: update charts.yml file
+        language: docker_image
+        entry: ghcr.io/alfresco/build-tools-gen-index-yml:OPSEXP-1908@sha256:3c3c6e9bb3df80be1f63bb972588685b5e297faeb7dc56324da5ad24da81294b
```

Comment:
update to master tag once https://github.com/Alfresco/alfresco-build-tools/pull/315 is merged